### ISSUE TITLE
Eliminating exchange votes without a blacklist

### ISSUE THREAD

**Issue by aiman** [consenlabs/carbonvote#4]
If the system detects conflicting votes from the same address, then just drop that address from participating in the vote.

The trade-off is that people won't be able to change their vote. But this may not be so bad, because it is consistent with how voting works in real elections. Also, it is relatively cheap to set up new rounds so they can be treated more as a snapshot in time.

If the "conflict vote" stats are interesting, you can provide a table listing the number of `YES` and `NO` transactions for each conflict address ranked by size.

**Comment by lgn21st** [comment]
I like the idea, but reconsider the recent event "The UK's EU referendum", I think there are a lots of leaving voters regret about the first action, so I think there is a big advance to vote through ethereum system that let people can be able to change they mind in last minute.

**Comment by aiman** [comment]
I agree that the flexibility is useful. In fact, there may be even better ways to exclude exchanges, such as using a method call instead of a transfer. There are surely many trade-offs.

**Comment by mrstormlars** [comment]
A new question has been set up for voting on carbonvote.com, but the blacklist has not been updated! E.g. kraken.com has 4 withdraw accounts in use, and only one is blacklisted... Needs to be fixed quickly! Don't know where to report...

**Comment by lgn21st** [comment]
@mrstormlars thank you for point out this, can you tell me how to find out all kraken withdraw address? I can fix it manually, then it will have new feature allow user submit exchange address directly at carbonvote.com

**Comment by mrstormlars** [comment]
You could start at the top of the Etherscan accounts list (sorted from highest ETH-amount): https://etherscan.io/accounts/1. Any known accounts are named in the list, so they are easy to spot. You will for example find Kraken_3 on the first page and Kraken_4 on the next.
